tests/blackbox_test.go

```diff
@@ -347,3 +358,16 @@ func outer(t *testing.T, test types.Test, negativeTests bool) error {
 		return fmt.Errorf("Expected failure and ignition succeeded")
 	}
 }
+
+// Remove a LUKS device
+func removeLuksDevice(deviceName string) error {
+	deviceExists, _ := util.PathExists(fmt.Sprintf("/dev/mapper/%s", deviceName))
```

Comment:
Why not check for errors here?

Reply:
I think I was worried that if the device did not show up it would error, but didn't think past that. as it would not matter if it did not exist (facepalm)